authui/src/authflowv2/botprotection/cloudflareTurnstile.ts

```diff
@@ -41,6 +41,10 @@ export class CloudflareTurnstileController extends Controller {
 
   connect() {
     window.turnstile.ready(() => {
+      // Do not render widget if any existing children
+      if (this.widgetTarget.childElementCount !== 0) {
+        return;
+      }
```

Comment:
<div class="bot-protection-widget-container-container"></div>
<div class="bot-protection-widget-container"></div>

Refer provider library reference on how to fit our lifecycle into provider library. You should perform proper cleanup in disconnect() https://developers.cloudflare.com/turnstile/get-started/client-side-rendering/#remove-a-widget